src/guide/component-provide-inject.md

```diff
@@ -49,7 +49,7 @@ app.component('todo-list-statistics', {
 })
 ```
 
-However, this won't work if we try to provide some component instance property here:
+However, this won't work if we try to provide same component instance property here:
```

Comment:
```suggestion
However, this won't work if we try to provide a component instance property here:
```

I don't think `same` is the intended word in this context.

Perhaps `a` would be clearer than `some`?

Reply:
I thought we were talking about the instance property of that particular component, therefore _same_, else it's correct.